pil2-stark/Makefile

```diff
@@ -0,0 +1,121 @@
+TARGET_BCT := bctree
+TARGET_STARKS_LIB := libstarks.a
+TARGET_SETUP := fflonkSetup
+
+BUILD_DIR := ./build
+LIB_DIR := ./lib
+SRC_DIRS := ./src
+SETUP_DIRS := ./src/rapidsnark
+SETUP_DPNDS_DIR := src/ffiasm
+WITNESS_LIB := ./witness_lib
+
+
+CXX := g++
+AS := nasm
+CXXFLAGS := -std=c++17 -Wall -pthread -flarge-source-files -Wno-unused-label -rdynamic #-Wfatal-errors
+LDFLAGS := -lprotobuf -lsodium -lgpr -lpthread -lpqxx -lpq -lgmp -lstdc++ -lgmpxx -lsecp256k1 -lcrypto -luuid -fopenmp -liomp5 
```

Comment:
@rickb80 should we remove some dependencies here?

Reply:
yes, I'll clean this.